test/arcade/core/util/ParameterTest.java

```diff
@@ -0,0 +1,150 @@
+package arcade.core.util;
+
+import org.junit.jupiter.api.Test;
+import ec.util.MersenneTwisterFast;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class ParameterTest {
+    @Test
+    public void testConstructor_populatesAttributes() {
+        MersenneTwisterFast random = new MersenneTwisterFast();
+        Parameter p = new Parameter(4.0, 0.25, false, random);
+
+        assertAll(
+                "Parameter",
```

Comment:
Thoughts on defaulting to having the `assertAll` description match the test name?

Reply:
It feels ike redundant info? What about just removing for obvious cases?